src/openings.md

```diff
@@ -2,7 +2,10 @@
 title: Openings
 description: Current Openings for positions on the Quilt Staff Team
 ---
-
+# Openings
+<p class="subtitle is-6">
+    Last Updated: {% edit_date %}
+</p>
```

Comment:
Again, you haven't done what I asked you to do here.

```html
<div class="media mb-5">
    <div class="media-content">
        <h1 class="title">
            {{ page.title }}
        </h1>
        <p class="subtitle is-6">
            Last Updated: {% edit_date %}
        </p>
    </div>
</div>

```

With a newline after it, since you need a new Markdown paragraph.